### DISCUSSION TITLE
See Nudge Logs in Jamf Pro

### DISCUSSION THREAD

**Discussion by bashkestrel** [dan-snelson/Nudge-Post-install#3] [General]
Hello, great work with the script here!  How are you seeing the logging from Nudge within the device records in Jamf Pro? An extension attribute?

**Comment by dan-snelson** [comment]
Happy Wednesday, @bashkestrel. 

The following may prove helpful:
https://macadmins.slack.com/archives/CDUU7DJQ2/p1629361231304700

**Reply by bashkestrel** [reply]
Thank you! Have you used that extension attribute successfully Dan?

**Reply by dan-snelson** [reply]
No, I have not. (We’re using Jamf Pro’s Patch Reporting for macOS Big Sur as confirmation that end-users are responding to “nudges.”)
